lua/onedark/theme.lua

```diff
@@ -354,14 +353,21 @@ function M.setup(config)
   }
 
   if config.hideInactiveStatusline then
-    local inactive = { style = "underline", bg = c.bg, fg = c.bg, sp = c.border }
+    local inactive = { style = "underline", bg = c.bg, fg = c.bg, sp = c.bg_visual }
 
-    -- StatusLineNC
+    -- StatusLine
     theme.base.StatusLineNC = inactive
 
-    -- LuaLine
-    for _, section in ipairs({ "a", "b", "c" }) do
-      theme.plugins["lualine_" .. section .. "_inactive"] = inactive
+    if vim.o.statusline ~= nil and string.find(vim.o.statusline, "lualine") then
+      -- Fix VertSplit & StatusLine crossover when lualine is active
+      -- https://github.com/ful1e5/onedark.nvim/issues/2
+      -- https://github.com/hoob3rt/lualine.nvim/issues/274
+      theme.base.StatusLine = { bg = c.bg }
+
+      -- LuaLine
+      for _, section in pairs({ "a", "b", "c" }) do
+        theme.plugins["lualine_" .. section .. "_inactive"] = inactive
+      end
```

Comment:
Was this the code that you changed in order to fix the `inactiveStatusLine` still showing a different background? 

I just tried this one my local and I'm still seeing the issue. I feel like it's somewhere else in my package. If I use `master` of your repository it works fine for me but as soon as I switch to mine the status line doesn't hide properly 😭